OpenRA.Mods.Common/Traits/Demolition.cs

```diff
@@ -36,6 +36,9 @@ class DemolitionInfo : ITraitInfo
 		[Desc("Duration of each flash.")]
 		public readonly int FlashDuration = 3;
 
+		[Desc("Only allow demolishing if order is a force fire.")]
+		public readonly bool RequiresForceFire = false;
```

Comment:
Can you please implement this as `TargetStances` and `ForceTargetStances` to match `Armament`?